src/mongo/db/audit/audit.cpp

```diff
@@ -1137,8 +1137,7 @@ class AuditPercona : public AuditInterface {
         BSONObjBuilder params;
         params << "ns" << toString(nss);
         params << "document" << doc;
-        params << "operation"
-               << "remove";
+        params << "operation" << "remove";
```

Comment:
We don't fix formatting if we did not change this line. Please remove unjustified formatting change.